src/lib/src/items-list.ts

```diff
@@ -36,17 +43,10 @@ export class ItemsList {
         this._selected = [];
     }
 
-    get value(): NgOption | NgOption[] {
-        if (this._multiple) {
-            return this._selected;
-        }
-        return this._selected[0];
-    }
-
     filter(term: string, filterFunc: FilterFunc) {
         this._markedItemIndex = -1;
         const filterFuncVal = filterFunc(term);
-        this.filteredItems = term ? this.items.filter(val => filterFuncVal(val)) : this.items;
+        this.filteredItems = term ? this.items.filter(filterFuncVal) : this.items;
```

Comment:
This is dangerous because you a losing original this scope, revert this or use filterFuncVal.bind(this)